docs/tutorials/contributing/strategy/writing_test_for_the_new_strategy.rst

```diff
@@ -65,66 +92,60 @@ All three of these functions can take an optional keyword argument
 
    This is equivalent to::
 
-    def test_effect_of_strategy(self):
-        P1 = axelrod.TitForTat() # Or whatever player is in your test class
-        P2 = axelrod.Player()
-        P2.history = []
-        P2.history = []
-        self.assertEqual(P1.strategy(P2), 'C')
+    P1 = axelrod.TitForTat() # Or whatever player is in your test class
+    P2 = axelrod.Player()
+    self.assertEqual(P1.strategy(P2), 'C')
 
-2. The member function :code:`markov_test` takes a list of four plays, each
+2. The member function :code:`second_play_test` takes a list of four plays, each
    following one round of CC, CD, DC, and DD respectively::
```

Comment:
`respectively. So for example here we test that Tit for tat will cooperate if and only if the opponent cooperates in the previous round::`